### PR TITLE
ling add citation for sbm paper

### PR DESCRIPTION
**Pull request by lanikaling**
Hi @sbillinge , the citation db is updated by adding the sbm paper. After the paper is accepted and published, I will update the db again by adding the doi and so on. Thx!

### CODE REVIEW

File: db/citations.yml [hunk 1]
```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
...
+llan;sbm;prm24:
+  ackno: ''
+  author:
+    - Ling Lan
+    - Qiang Du
+    - Simon J. L. Billinge
+  doi: ''
+  entrytype: article
+  facility: ''
+  grant: DE-SC0024141
```
Comment (sbillinge) [inline db/citations.yml:8467]: doeneutron23 for grant

File: db/citations.yml [hunk 2]
```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
...
+  journal: Physical Review Materials
+  month: tbd
+  nb: ''
+  note: submitted
```
Comment (sbillinge) [inline db/citations.yml:8471]: put the arxiv reference here when we have it

File: db/citations.yml [hunk 3]
```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
   wwwemail: ''
   wwwpub: ''
   year: '2022'
+llan;sbm;prm24:
+  ackno: ''
+  author:
+    - Ling Lan
+    - Qiang Du
+    - Simon J. L. Billinge
+  doi: ''
```
Comment (sbillinge) [inline db/citations.yml:8464]: put the arxiv doi here when we have it

File: db/citations.yml [hunk 4]
```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
...
+  grant: DE-SC0024141
+  journal: Physical Review Materials
+  month: tbd
+  nb: ''
+  note: submitted
+  number: ''
+  optannote: ''
+  optwwwlanl: ''
+  pages: ''
+  synopsis: A quantitative measure of symmetry breaking is introduced that allows the quantification
```
Comment (sbillinge) [inline db/citations.yml:8476]: synopsis is way shorter.  Something like `introduces the continuous symmetry breaking measure based on Jensen-Shannon divergence" or something.

What you have written could go in under a key of `professional_summary:`

File: db/citations.yml [hunk 5]
```diff
@@ -8455,6 +8455,40 @@ llan;RobustnessTestSpacegroupMining22:
...
+   Software code is presented that carries the calculations out numer- ically using Monte Carlo 
+   methods. The behavior of this symmetry breaking measure is tested for various cases including 
+   finite size crystallites (where the surfaces break the crystallographic symmetry), atomic 
+   displacements from high symmetry positions, and collective motions of atoms due to rotations of 
+   rigid octahedra. The approach provides a powerful tool for assessing local symmetry breaking 
+   and offers new insights that can help researchers understand how different structural distortions 
+   affect different symmetry operations.
+  tags: symmetry_breaking
+  title: A continuous symmetry breaking measure for finite clusters using Jensen-Shannon divergence
+  url: ''
```
Comment (sbillinge) [inline db/citations.yml:8489]: put arxiv url here

**Review by sbillinge — COMMENTED** [review]
please see comments.  Let's get the arxiv reference and update under this pr to avoid extra work....

File: db/citations.yml (hunk 1 above)
Comment (lanikaling) [reply db/citations.yml:8467]: This is different from the paper acknowledgment. In the paper, we used DE-SC0024141 which you have confirmed twice. Do we want to change the paper acknowledgment?

Comment (sbillinge) [reply db/citations.yml]: no.  What is needed here is the regolith database ID of the grant.

**Comment by lanikaling** [comment]
Hi @sbillinge, thank you for all your comments. All of them are resolved and this PR should be good to go.

**Comment by sbillinge** [comment]
Thanks Ling!

**Pull request merged**